tests/JustSaying.IntegrationTests/Fluent/AwsTools/Approvals/WhenEnsuringATopicExistsPolicyIsCreated.Then_The_Iam_Policy_Is_Created.approved.txt

```diff
@@ -8,10 +8,10 @@
                 "AWS" : "*"
             },
             "Action"    : "sqs:SendMessage",
-            "Resource"  : "",
+            "Resource"  : "arn:aws:sqs:us-east-1:000000000000:<unique-name>",
             "Condition" : {
                 "ArnLike" : {
-                    "aws:SourceArn" : "*"
+                    "aws:SourceArn" : "arn:aws:sns:us-east-1:000000000000:<unique-name>-topic"
```

Comment:
I could be wrong, but I _think_ the us-east-1 here is an localstack thing

Reply:
It is, localstack lets you override it by specifying a `DEFAULT_REGION` env var

Reply:
I'll give it a go with the region set just to make sure nothing weird is happening with the region being sent

Reply:
Yeah, looks good

<img width="513" alt="image" src="https://user-images.githubusercontent.com/2156707/229375533-cc402679-7e26-474a-8f42-314d2afab717.png">